recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: pip install --no-deps .
```

Comment:
@minrk just curious, what changed in this version that is not causing the same failure we saw in the previous one using `pip`?